src/components/graph/util/model-functions.js

```diff
@@ -73,7 +74,7 @@ export function isNodeReadOnly(node) {
 export function isNodeBuilt(node) {
     if (!node) return false;
     if (node.type === 'ROOT') return true;
-    return node.data?.buildStatus === 'BUILT';
+    return node.data?.buildStatus.startsWith('BUILT');
```

Comment:
Maybe ?.startsWith('BUILT') for safety ?

Reply:
Done